src/scss/pages/_landing.scss

```diff
@@ -178,6 +172,7 @@
                 @include mediaMin(xs){
```

Comment:
xs is default breakpoint . starting form opx  . xs is default why you using media query this breakpoint